### DISCUSSION TITLE
Which os or distro is this?

### DISCUSSION THREAD

**Discussion by Chandraprakash-Darji** [NvChad/NvChad#1787] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/93640141/220257784-033408d7-388a-4b4a-9987-6f3c7a61f678.png)
In the readme which os is this?

**Comment by siduck** [comment, marked as answer]
hi @Chandraprakash-Darji that was void. Also the distro doesnt matter. I customized bspwm all on my own from scratch. check my dotfiles repo

**Comment by Chandraprakash-Darji** [comment]
@siduck thanks for help out.
